### ISSUE TITLE
i18n of checkwiki_errors.py

### ISSUE THREAD

**Issue by edgarskos** [matejsuchanek/pywikibot-scripts#9]
Maybe we could add some dict of localized edit summaries? Of course, I can go and translate them in my copy of this script, but if there are some major changes in script and I would like to follow them, I would need to do that again. That wouldn't be too cool :)

**Comment by matejsuchanek** [comment]
Certainly, I will add localisation soon. But first, I need to change the organisation (ie. the tree) to make it more usable itself (or for instance as much usable for `user_fixes.py` as possible).
